### PR TITLE
Examples: clean up

### PR DESCRIPTION
**Pull request by WestLangley**
These examples were all static scenes, with an animation loop.

This PR adds controls damping, since it is free in this case. Plus, some clean up.

### CODE REVIEW

File: examples/webgl_loader_vrml.html [hunk 1]
```diff
@@ -31,23 +31,23 @@
...
 				scene.add( camera );
 
 				// light
 
 				var dirLight = new THREE.DirectionalLight( 0xffffff );
-				dirLight.position.set( 200, 200, 1000 ).normalize();
+				dirLight.position.set( 200, 200, 1000 );
 
 				camera.add( dirLight );
 				camera.add( dirLight.target );
```
Comment (WestLangley) [inline examples/webgl_loader_vrml.html:53]: @Mugen87 For a later PR, this pattern occurs in several examples. I'd remove it... Perhaps replace it with simply `camera.add( pointLight )`?

Comment (Mugen87) [reply examples/webgl_loader_vrml.html:53]: I have no preference there. I just think using directional lights are fine. The call of `normalize()` however is indeed confusing and should be removed 👍

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Comment by WestLangley** [comment]
@Mugen87 Is the `VRMLoader` example supposed to animate? It is a collection of skinned meshes. It is also throwing `GLTFLoader` errors.

/ping @takahirox

**Comment by takahirox** [comment]
No, the example should just render a model without animation. I'll look into the errors later.

**Pull request merged**